feeds/capcotentin.fr.dmfr.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://dmfr.transit.land/json-schema/dmfr.schema-v0.5.0.json",
+  "feeds": [
+    {
+      "id": "f-cap~cotentin~transit",
```

Comment:
Please add a `tags.notes` property on the feed describing the situation.

Reply:
It would probably also be useful to add a `tags.notes` on to the `f-sp9x-normandie` record as well.

Reply:
Finally, it would also be useful to add to this feed: `tags.exclude_from_global_query=true` The Transitland stack will then import both static feeds, but exclude the `f-cap~cotentin~transit` routes/stops/etc from global searches and tiles.

Reply:
I think I did what you requested but I also feel I messed up with the tags. I'm not sure I understood properly their structure. Could you have a look @drewda? Thanks for your review!

Reply:
Thanks for all your edits. All looks good, other than that the `true` value needed to be in quotation marks.